homeassistant/components/analytics/analytics.py

```diff
@@ -206,16 +216,37 @@ async def send_analytics(self, _: datetime | None = None) -> None:
         if self.preferences.get(ATTR_USAGE, False) or self.preferences.get(
             ATTR_STATISTICS, False
         ):
+            ent_reg = er.async_get(self.hass)
+
+            try:
+                yaml_configuration = await conf_util.async_hass_config_yaml(self.hass)
+            except HomeAssistantError as err:
+                LOGGER.error(err)
+                return
+
+            configuration_set = set(yaml_configuration.keys())
```

Comment:
`.keys()` is the default iterator so can be omitted
```suggestion
            configuration_set = set(yaml_configuration)
```